.github/workflows/package-cli.yml

```diff
@@ -83,30 +95,43 @@ jobs:
         run: git apply --allow-empty ${{ runner.temp }}/version.patch/version.patch
         timeout-minutes: 1
 
-      # Install fuse for building the CLI (libparsec includes platform_mountpoint)
-      - name: Install fuse
+      - name: Use Cross
         shell: bash
         run: |
-          until sudo apt-get -y install fuse3 libfuse3-dev; do
-            echo "Failed to install APT package, retrying...";
-          done
+          dir="$RUNNER_TEMP/cross-download"
+          mkdir "$dir"
+          echo "$dir" >> $GITHUB_PATH
+          cd "$dir"
+          curl -LO "https://github.com/cross-rs/cross/releases/download/${{ env.cross-version }}/cross-x86_64-unknown-linux-musl.tar.gz"
+          tar xf cross-x86_64-unknown-linux-musl.tar.gz
         timeout-minutes: 5
 
       - name: Build CLI
-        run: cargo build -p parsec-cli --release
-        timeout-minutes: 10
+        run: >-
+          cross build -p parsec-cli
+          --release
```

Comment:
pass flags with `$(make.py cli-release-libparsec-cargo-flags)`

The `--target` should be kept here, and vendored-openssl passed through the `MAYBE_FORCE_VENDORED_OPENSSL` environ variable